### DISCUSSION TITLE
How do I get a list of all errors while upgrading to Next 15?

### DISCUSSION THREAD

**Discussion by aquibbaig** [vercel/next.js#73132] [Help] [answered]
### Summary

I am upgrading from next 14 -> 15 in my project. What's really weird is how I am getting selective code splitted errors (when I go to a place, the error shows up on the terminal). 
I need to do a lot of migrations from params -> async params (searchParams). 

It does not sound feasible going page by page for fixing errors like these (which is manual and error-prone). Is there a command that can list all these errors altogether so that I can fix them? `pnpm build` does not list them as errors.

P.S. - I do not want to use the codemod.

**Comment by icyJoseph** [comment, marked as answer]
I know you said, you don't want to, but the codemod worked great for me.

That being said, what kind of errors are we talking about? Is it the `Promise` changes?

Maybe try this, if its TS errors: https://github.com/vercel/next.js/discussions/48943#discussioncomment-9138317

```json
  "build:all-ts-errors": "__NEXT_TEST_MODE=true yarn build"
```

Of course, fit it to pnpm
